templates/less/components/form.less

```diff
@@ -27,43 +28,58 @@
     padding-bottom: 10px;
   }
 
-  .icon(@fa-icon) {
-    content: @fa-icon;
-    .fa();
+  .leftIcon() {
+    z-index: 1;
+    top: 18px;
+    left: 15px;
   }
 
-  .label(@fa-icon) {
-    .icon(@fa-icon);
-    color: #B5B1A6;
-    font-family: FontAwesome;
-    position: absolute;
-    top: 16px;
-    left: 5%;
-    font-size: 18px;
+  .icon-label-username,
+  .icon-label-password,
+  .icon-label-email {
+    position: relative;
   }
 
   .icon-label-username:before {
     .label(@fa-username);
+    .leftIcon()
   }
   .icon-label-password:before {
     .label(@fa-password);
+    .leftIcon()
   }
   .icon-label-email:before {
     .label(@fa-email);
+    .leftIcon()
   }
-  .icon-label-username,
-  .icon-label-password,
-  .icon-label-email {
-    position: relative;
+
+  .icon-label-error:after {
+    .label(@fa-warn);
+    right: 10px;
+    top: 17px;
+    color: #e7518f;
+  }
+  .icon-label-valid:after {
+    .label(@fa-check);
+    right: 10px;
+    top: 17px;
+    color: #53C0A3;
   }
 }
 .has-error {
-  border: 1px solid #E95B95;
+  border: 1px solid #E95B95 !important;
```

Comment:
pretty sure you don't. This is a sign that the classes you're using are mutually exclusive and you're "pushing one of them to win the CSS war". is-valid and has-error should be mutually exclusive, so either the base grey gets overwritten with green, or it gets overwritten with red. We should not need `!important`